src/query/ast/src/parser/expr.rs

```diff
@@ -1443,7 +1452,7 @@ pub fn json_op(i: Input) -> IResult<JsonOperator> {
         value(JsonOperator::LongArrow, rule! { "->>" }),
         value(JsonOperator::HashArrow, rule! { "#>" }),
         value(JsonOperator::HashLongArrow, rule! { "#>>" }),
-        value(JsonOperator::Question, rule! { "?" }),
+        // value(JsonOperator::Question, rule! { "?" }),
```

Comment:
We can keep `?` as `JsonOperator::Question`, and convert to `Placeholder` when there are no left and right expressions. Like `+` can both parsed as `BinaryOperator::Plus` and `UnaryOperator::Plus`.